tripal_chado/includes/tripal_chado.field_storage.inc

```diff
@@ -43,6 +43,7 @@ function tripal_chado_field_storage_write($entity_type, $entity, $op, $fields) {
 
   // Convert the fields into a key/value list of fields and their values.
   $field_vals = tripal_chado_field_storage_write_merge_fields($fields, $entity_type, $entity);
+// dpm($field_vals);
```

Comment:
can we remove this dpm?  even commented out it drives me crazy sometimes searching for my own dpms.